tests/unit/core/api/test_search.js

```diff
@@ -0,0 +1,301 @@
+/* global window */
+import config from 'config';
+
+import search from 'core/api/search';
+import {
+  ADDON_TYPE_EXTENSION,
+  ADDON_TYPE_THEME,
+  CLIENT_APP_ANDROID,
+  CLIENT_APP_FIREFOX,
+} from 'core/constants';
+import { parsePage } from 'core/utils';
+import { createApiResponse } from 'tests/unit/core/api/helpers';
+import { dispatchSignInActions } from 'tests/unit/amo/helpers';
+import { unexpectedSuccess } from 'tests/unit/helpers';
+
+
+describe(__filename, () => {
+  let api;
+  let mockWindow;
+  const apiHost = config.get('apiHost');
+  const esrFirefoxUA = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv:52.2.1) Gecko/20100101 Firefox/52.2.1';
```

Comment:
I can add these to the existing userAgent helpers actually, let me know and I can do that in a little fix-up commit.